docs/4-cloud-computing/4.3.3-vmss.md

```diff
@@ -47,30 +47,21 @@ app.get('/', function (req, res) {
 app.listen(3000, function () {
   console.log('Test app listening on port 3000!')
 })
-EOF
 ```
 
 7. Build the Node.js app locally first. Launch it locally and make sure it works.
 
-```bash
-cd /Users/YOUR_NAME/webapp
-npm init --yes
-npm install express -y
-```
-
 8. Upload the Node.js app to Azure file share we created in step 5.
 
 ```bash
-az storage file upload-batch \
-  --account-key $STORAGE_KEY \
-  --account-name $STORAGE_ACCOUNT \
-  --destination $SHARE_NAME --source /Users/YOUR_NAME/webapp
+az storage file upload-batch -h
 ```
 
 9. Update the cloud-init script for VMSS. It now will mount Azure Files share containing our Node.js app and launch from that mount. This way multiple VMs can serve the same content from a central location.
 
-```bash
-cat <<EOF > /Users/YOUR_NAME/cloud-init-vmss.txt
+!> The below won't work unless you have properly setting environment variables for STORAGE_ACCOUNT, SHARE_NAME, and STORAGE_KEY.
```

Comment:
Kind of. Even with those env variables set this will not work. It worked before because we piped a heredoc interpreted in the context of the shell into a file. 

Either leave this part as it was so with the hint. Or advice campers to substitute those values in themselves.

Reply:
Consider it done! I understand what you mean. I should have caught that. I think we may have done that in another section. I'll have to look.